### PR TITLE
chore: Log YAML to V7, instead of V4

### PR DESCRIPTION
**Pull request by karlkfi**
- Rename "resource" to "object" in mutation code.
  Resource is a REST interface. Object is an instance of a Resource.

### CODE REVIEW

File: pkg/apply/mutator/apply_time_mutator.go [hunk 1]
```diff
@@ -134,25 +136,26 @@ func (atm *ApplyTimeMutator) Mutate(ctx context.Context, obj *unstructured.Unstr
...
 
 		mutated = true
-		reason = fmt.Sprintf("resource contained annotation: %s", mutation.Annotation)
+		reason = fmt.Sprintf("object contained annotation: %s", mutation.Annotation)
 	}
 
 	if mutated {
-		klog.V(4).Infof("mutated target resource (%s):\n%s", targetRef, object.YamlStringer{O: obj})
+		klog.V(4).Infof("mutated target object: %s", targetRef)
+		klog.V(7).Infof("mutated target object YAML:\n%s", object.YamlStringer{O: obj})
```
Comment (Liujingfang1) [inline pkg/apply/mutator/apply_time_mutator.go:151]: I'm curious why logging the whole object yaml instead of the fields being mutated.

Comment (karlkfi) [reply pkg/apply/mutator/apply_time_mutator.go:151]: It's mostly just for debugging when the JSONPath fails to identify the field you wanted.

Comment (Liujingfang1) [reply pkg/apply/mutator/apply_time_mutator.go:151]: Ack

**Pull request merged**